src/modules/compass/Compass.js

```diff
@@ -229,6 +234,75 @@ const Compass = () => {
     );
   };
 
+  const renderMeasurementTemplates = () => {
+    return (
+      <React.Fragment>
+        <View style={{borderBottomWidth: 1}}>
+          <ListItem containerStyle={commonStyles.listItem}>
+            <ListItem.Content containerStyle={commonStyles.listItem}>
```

Comment:
I don't think the style needs be on the ListItem.Content since it's on the ListItem above